spdx/parsers/lexers/tagvalue.py

```diff
@@ -95,7 +96,11 @@ def t_text_end(self, t):
         t.type = 'TEXT'
         t.value = t.lexer.lexdata[
             t.lexer.text_start:t.lexer.lexpos].strip()
-        t.lexer.lineno += t.value.count('\n')
+        try :
+        	unicode
+        	t.lexer.lineno += t.value.count('\n'.encode("utf-8"))
```

Comment:
If this is needed to pass the test, then may be the tests need to be updated instead?